navigation_map_path : /home/movel/.config/movel/maps/nav/
localization_map_path : /home/movel/.config/movel/maps/
line_width : 1